code/src/entrance.c

```diff
@@ -23,6 +23,7 @@ void Scene_Init(void) {
     memcpy(&gSceneTable[13], gSettingsContext.ganonsCastleDungeonMode          == DUNGEONMODE_MQ ? &gMQDungeonSceneTable[13] : &gDungeonSceneTable[13], sizeof(Scene));
 
     gRestrictionFlags[72].flags2 = 0; // Allows warp songs in GTG
+    gRestrictionFlags[93].flags2 = 0; // Allows warp songs in Windmill / Dampe's grave
```

Comment:
I do think it's best to leave this out for now to preserve the trick with dampe's race. Can you either comment it out for now or just remove it?

Reply:
Done. I commented it so it can be reenabled in the future without looking for that "93" again